app/Platform/Enums/GameTitleRegion.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Enums;
+
+enum GameTitleRegion: string
+{
+    case Africa = 'af';
+    case Australia = 'au';
+    case Brazil = 'br';
+    case China = 'cn';
+    case Europe = 'eu';
+    case India = 'in';
+    case Japan = 'jp';
+    case LatinAmerica = 'latam';
+    case MiddleEast = 'me';
+    case NorthAmerica = 'na';
+    case Russia = 'ru';
+    case SoutheastAsia = 'sea';
+    case SouthKorea = 'kr';
+    case TaiwanHongKong = 'tw_hk';
```

Comment:
I'm curious how you decided on these regions.

NA/EU/JP seem obvious (I might even argue US over NA), but are there really that many releases unique to India and the Middle East? And what's left for Southeast Asia after you take out China, Korea, Taiwan and Hong Kong?

Reply:
This is probably way too wide of a net.

In latest, I've renamed this enum to `GameReleaseRegion` and modeled it after IGDB's region choices:
<img width="186" alt="Screenshot 2025-05-20 at 4 34 52 PM" src="https://github.com/user-attachments/assets/ad415ea3-d53a-4708-9016-43f21c3c09b3" />